client/src/modules/roles/modal/userRole.html

```diff
@@ -23,7 +23,7 @@ <h4><span translate>FORM.LABELS.ROLES_FOR</span>
 
 <div class="modal-footer">
   <div class=" pull-right">
-  <button class='btn inverse' ng-click='UsersRolesCtrl.close()'>Fermer</button>
+  <span class='btn inverse' ng-click='UsersRolesCtrl.close()'>Fermer</span>
```

Comment:
Oh - this should be translated.  I never noticed it before.  Could you make this a button as well?